src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AwaitExpressionSpiller.cs

```diff
@@ -372,34 +372,42 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
 
                     case BoundKind.FieldAccess:
                         var field = (BoundFieldAccess)expression;
-                        if (field.FieldSymbol.IsReadOnly)
+                        var fiedlSymbol = field.FieldSymbol;
```

Comment:
uugh

---
In reply to: [127517734](https://github.com/dotnet/roslyn/pull/20883#discussion_r127517734) [](ancestors = 127517734)